### ISSUE TITLE
Crashes Spotify integration 

### ISSUE THREAD

**Issue by zboersen** [jonathanrobichaud4/Spotify-custom-scan-interval#1]
Very handy integration. Unfortunately it crashes the Spotify integration after about 24 hours. 

Maybe it is not compatible with later versions.

**Comment by jonathanrobichaud4** [comment]
I would say don't use this integration anymore. I made this as a temp fix but on the topic on the HA forums someone used node red to call homeassistant.update_entity every couple of seconds. This was because I tried to use spotcast with this integration and it did not work. I can share the node red flow if you want?

**Comment by zboersen** [comment]
Oh, okay. Thanks for the info!

No that is not necessary. 



Met vriendelijke groet,

Zoë Boersen

Van: Jonathan Robichaud
Verzonden: vrijdag 26 november 2021 14:28
Aan: jonathanrobichaud4/Spotify-custom-scan-interval
CC: zboersen; Author
Onderwerp: Re: [jonathanrobichaud4/Spotify-custom-scan-interval] Crashes Spotify integration (Issue #1)

I would say don't use this integration anymore. I made this as a temp fix but on the topic on the HA forums someone used node red to call homeassistant.update_entity every couple of seconds. This was because I tried to use spotcast with this integration and it did not work. I can share the node red flow if you want?
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
